twig/BS3/fields/static.twig

```diff
@@ -1,4 +1,23 @@
 {% macro render(field) %}
+    {% if field.label is not null %}
```

Comment:
What is all of this for?  Static fields are display only.

Reply:
Static fields don't have an input, and so the label is an "orphaned" label.  This is to make it so that they just get a strong text instead of a label. Visually stays the same, for screen readers it doesn't announce an input that doesn't exist.

Reply:
Is there a universal label that can apply to both an input and a div?  You can add `contenteditable="true"` to a div and the content becomes editable, and it's not an input, select, nor textarea.

Reply:
cool!